### PR TITLE
Fix Yast2::ServiceWidget to not show "On Boot" option

### PR DESCRIPTION
**Pull request by imobachgs**

### CODE REVIEW

File: library/systemd/test/yast2/service_widget_test.rb [hunk 1]
```diff
@@ -68,6 +85,51 @@
 
       expect(autostart_selector).to_not be_nil
     end
+
+    describe "start mode selector" do
+      let(:autostart_selector) { find_term(subject.content, :ComboBox, :service_widget_autostart) }
+      let(:items) { autostart_selector.params.last }
+
+      before do
+        allow(service).to receive(:support_start_on_demand?).and_return(on_demand)
```
Comment (joseivanlopez) [inline library/systemd/test/yast2/service_widget_test.rb]: NP: this before is not needed.

File: library/systemd/test/yast2/service_widget_test.rb [hunk 2]
```diff
@@ -68,6 +85,51 @@
...
+
+        it "does not include an on demand option" do
+          expect(has_item?(autostart_selector, :service_widget_autostart_on_demand)).to eq(false)
+        end
+      end
+
+      context "when the service supports to be started on boot" do
+        let(:on_boot) { true }
+
+        it "includes a on boot option" do
```
Comment (joseivanlopez) [inline library/systemd/test/yast2/service_widget_test.rb]: NP: typo "an on boot"

File: library/systemd/test/yast2/service_widget_test.rb [hunk 3]
```diff
@@ -68,6 +85,51 @@
...
+
+      context "when the service does not support to be started on boot" do
+        let(:on_boot) { false }
+
+        it "does not include an on boot option" do
+          expect(has_item?(autostart_selector, :service_widget_autostart_on_boot)).to eq(false)
+        end
+      end
+
+      it "includes an manual option" do
```
Comment (joseivanlopez) [inline library/systemd/test/yast2/service_widget_test.rb]: NP: typo "an manual"

File: package/yast2.changes [hunk 4]
```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Fri Aug 24 13:36:42 UTC 2018 - igonzalezsosa@suse.com
+
+- Fix the Yast2::ServiceWidget to not should "On Boot" option
```
Comment (joseivanlopez) [inline package/yast2.changes]: "to not should" or "to not show"

**Comment by lslezak** [comment]
:heavy_check_mark: Created OBS submit [request #631389](https://build.opensuse.org/request/show/631389).

**Pull request merged**